### ISSUE TITLE
Add requirements.txt with linters/formatters that can be installed from PyPI

### ISSUE THREAD

**Issue by abelsiqueira** [nlesc-recruit/cudawrappers#90]
> We could put the things you can get from pypi into a requirements.txt
> 
> _Originally posted by @jspaaks in https://github.com/nlesc-recruit/CUDA-wrappers/pull/77#discussion_r816741332_

**Comment by fdiblen** [comment]
Also update the developer documentation.

**Comment by jspaaks** [comment]
Alternatively, maybe we can just add a note to the README.dev.md, something along th elines of 

```
# install cppcheck, clang-tidy, clang-format from your package manager
sudo apt install cppcheck clang-format clang-tidy

# install python3 and pip
sudo apt install python3 python3-pip

# install flawfinder, cmake-format, cmake-lint from PyPI using pip
python3 -m pip install flawfinder cmake-format cmake-lint 
```

**Comment by fdiblen** [comment]
> Alternatively, maybe we can just add a note to the README.dev.md, something along th elines of
> 
> ```
> # install cppcheck, clang-tidy, clang-format from your package manager
> sudo apt install cppcheck clang-format clang-tidy
> 
> # install python3 and pip
> sudo apt install python3 python3-pip
> 
> # install flawfinder, cmake-format, cmake-lint from PyPI using pip
> python3 -m pip install flawfinder cmake-format cmake-lint 
> ```

I think this will be cleaner.

**Comment by fdiblen** [comment]
fixed by #105
